src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -0,0 +1,104 @@
+import React, { useContext, useState } from "react";
+import { ConfigContext } from "Context";
+import { useAsync } from "react-use";
+import Loader from "components/Loader";
+import { FormattedMessage, useIntl } from "react-intl";
+import { v4 as uuidv4 } from "uuid";
+import {
+  ButtonGroup,
+  Heading,
+  LinkButton,
+} from "@utrecht/component-library-react";
+import { get, put } from "../../api";
+import PropTypes from "prop-types";
+
+const propTypes = PropTypes.shape({
+  heading: PropTypes.string,
+  headingLevel: PropTypes.number,
+  onLanguageChange: PropTypes.func,
+});
+
+const LanguageSelection = (props) => {
```

Comment:
destructuring the props in the callback allows you to provide default values for optional props, so you don't need to specify `defaultProps`:

```jsx
const LanguageSelection = ({
  heading=<FormattedMessage description="Language selection heading" defaultMessage="Choose language" />,
  headingLevel=2,
  onLanguageChange=console.log,
}) => {
```

the `onLanguageChange` prop will probably be replaced with something we get from the config context instead, so I wouldn't worry about that one and/or the default value too much now.